### PR TITLE
docs: misc changes while translating (smoe:docs29_mac_1)

### PR DESCRIPTION
**Pull request by smoe**
Submitted against 2.9

### CODE REVIEW

File: docs/src/gui/qtvcp-widgets.adoc [hunk 1]
```diff
@@ -857,7 +793,10 @@ For `GcodeDisplay` widget's _default G-code lexer_:
 * *styleColor6 = AxisValue*: Values following XYZABCUVW
 * *styleColor7 = OtherValue*: Values following EFHIJKDQLRPST$
 
-NOTE: For comments, the "OtherValue" color (Color 5) can be used to highlight "print," "debug," "msg," "logopen," "logappend," "logclose" "log," "pyrun," "pyreload" "abort," "probeopen" "probeclose" inside of a parenthesis comment in a line of G-code. As well as "py," if a line that starts with ";py,". Examples: (print, text), (log, text), (msg, text), or (debug, text). Only the last of the examples will be highlighted if there are more than one on the same line.
+NOTE: For comments, the "OtherValue" color (Color 5) can be used to highlight "print," "debug," "msg," "logopen," "logappend," "logclose" "log," "pyrun," "pyreload" "abort," "probeopen" "probeclose" inside of a parenthesis comment in a line of G-code.
+As well as "py," if a line that starts with ";py,".
+Examples: (print, text), (log, text), (msg, text), or (debug, text).
+Only the last of the examples will be highlighted if there are more than one on the same line.
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:799]: This you pulled out of the NOTE?

File: src/hal/components/plasmac.comp [hunk 2]
```diff
@@ -111,11 +111,11 @@ pin in  bit     probe_test                  "probe test only";
 pin in  bit     program_is_idle             "program is idle, connect to halui.program.is-idle";
 pin in  bit     program_is_paused           "program is paused, connect to halui.program.is-paused";
 pin in  bit     program_is_running          "program is running, connect to halui.program.is-running";
-pin in  float   puddle_jump_delay           "delay move from pierce height to cut height (seconds), leave disconnected if not required";
-pin in  float   puddle_jump_height          "puddle jump height (percentage of pierce height), leave disconnected if not required";
+pin in  float   puddle_jump_delay           "Relay move from pierce height to cut height (seconds), leave disconnected if not required.";
```
Comment (hansu) [inline src/hal/components/plasmac.comp:114]:
```suggestion
pin in  float   puddle_jump_delay           "Delay move from pierce height to cut height (seconds), leave disconnected if not required.";
```

File: docs/src/gui/qtvcp-widgets.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/gui/qtvcp-widgets.adoc:799]: Not intentionally.

Comment (hansu) [reply docs/src/gui/qtvcp-widgets.adoc:799]: I haven't checked if it's really the case, it only looked so in the GitHub diff rendering...

**Comment by smoe** [comment]
@hansu  - removed conflicts.

File: src/hal/components/plasmac.comp (hunk 2 above)
Comment (hansu) [reply src/hal/components/plasmac.comp:114]: Not resolved  yet...

Comment (hansu) [reply src/hal/components/plasmac.comp:114]: Not resolved yet...

**Review by hansu — CHANGES_REQUESTED** [review]

**Comment by smoe** [comment]
Done. Thank you for spotting the Delay->Relay.

**Review by hansu — APPROVED** [review]

**Pull request merged**